static/game/powerups/quickshot.js

```diff
@@ -2,7 +2,7 @@ import { PowerUp } from "../powerup.js";
 import { Vector } from "../../modules/vector.js";
 import { Creature } from "../creature.js";
 
-const MAX_BULLET_SPEED = 50;
+const MAX_BULLET_SPEED = 24;
```

Comment:
width and height of a bullet is actually also 24, so we can add the radius of the smallest bullet to this. the new speed of light could be 24 + 12 - 1 = 35 (the minus one is for safety so it can't jump exactly half)

we could also just test this empirically

Reply:
24 is the maximum I found experimentally
We can try to increase it later on by using the bullet's velocity with collision detection (marking it as "fast" like box2D is a good solution but it's one for another PR.